src/util/get-all-keys.js

```diff
@@ -0,0 +1,7 @@
+
+export default function keys(obj = {}, { enumerable = false } = {}) {
+  const listOfKeys = Object[enumerable ? 'getOwnPropertyNames' : 'keys'](obj);
```

Comment:
Might be good to invert the clause for readability so your mind doesn't have to invert the variable value.